src/main/java/com/msalvatore/graphqlplayground/entity/Vehicle.java

```diff
@@ -0,0 +1,76 @@
+package com.msalvatore.graphqlplayground.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Entity
```

Comment:
Hmm, the table name should probably be `VEHICLES`. Will update. Maybe I should have reviewed this better before opening the PR heh.